pkgs/checks/lib/src/extensions/math.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:checks/context.dart';
+
+extension NumChecks on Check<num> {
+  void isGreaterThan(num other) {
+    context.expect(() => ['is greater than ${literal(other)}'], (actual) {
+      if (actual > other) return null;
+      return Rejection(
+          actual: literal(actual),
+          which: ['Is not greater than ${literal(other)}']);
+    });
+  }
+
+  void isLessThan(num other) {
+    context.expect(() => ['is less than ${literal(other)}'], (actual) {
+      if (actual < other) return null;
+      return Rejection(
+          actual: literal(actual),
+          which: ['Is not less than ${literal(other)}']);
+    });
+  }
```

Comment:
(Probably also want `.isFinite`, `.isNaN` and `.abs()` (as `=> has((n) => n.abs(), '.abs()')`).
Maybe even the `floor`/`ceil`/`round` functions.